Count Occurrences/README.markdown

```diff
@@ -22,7 +22,7 @@ The trick is to use two binary searches, one to find where the `3`s start (the l
 In code this looks as follows:
 
 ```swift
-func countOccurrences<T: Comparable>(of key: T, in array: [T]) -> Int {
+func countOccurrences<T: Comparable>(of key: T, in a: [T]) -> Int {
```

Comment:
`array` would be preferable here. I would refer to some of the principles in the Swift API design [guidelines](https://swift.org/documentation/api-design-guidelines/) for this:

> Clarity is more important than brevity. Although Swift code can be compact, it is a non-goal to enable the smallest possible code with the fewest characters. Brevity in Swift code, where it occurs, is a side-effect of the strong type system and features that naturally reduce boilerplate.

Reply:
The README is not accurate with the actual code. All the `a` references should actually be `array`.

Reply:
Agree. Array is clearer. Just exchanged the renaming.